test/resource_feedback_test.rb

```diff
@@ -0,0 +1,28 @@
+class ResourceFeedbackTest < Test::Unit::TestCase
+  def test_save_with_resource_feedbacks_endpoint
+    body = { resource_feedback: {} }.to_json
+    fake 'resource_feedbacks', method: :post, body: body
+    ShopifyAPI::ResourceFeedback.new.save
+    assert_request_body body
+  end
+
+  def test_save_with_product_id_resource_feedbacks_endpoint
+    body = { resource_feedback: {} }.to_json
+    fake 'products/42/resource_feedbacks', method: :post, body: body
+    ShopifyAPI::ResourceFeedback.new(product_id: 42).save
+    assert_request_body body
+  end
+
+  def test_save_notify_via_bugsnag_when_already_persisted
```

Comment:
Good catch! If they are 3P, the exception may or may not be show on bugsnag. I updated the function name so it's more correct.